src/Microsoft.AspNetCore.Diagnostics.Elm/Microsoft.AspNetCore.Diagnostics.Elm.csproj

```diff
@@ -2,8 +2,10 @@
 
   <PropertyGroup>
     <Description>ASP.NET Core Error Logging Middleware (ELM) to capture and display request logs.</Description>
-    <VersionPrefix Condition="'$(ExperimentalProjectVersionPrefix)' != ''">$(ExperimentalProjectVersionPrefix)</VersionPrefix>
-    <VerifyVersion Condition="'$(ExperimentalProjectVersionPrefix)' != ''">false</VerifyVersion>
+    <VersionPrefix Condition="'$(ExperimentalVersionPrefix)' != ''">$(ExperimentalProjectVersionPrefix)</VersionPrefix>
```

Comment:
Can we remove the conditional check?

Reply:
I used the pattern intentionally. If someone were to unset ExperimentalVersionPrefix, NuGet will default to producing a package version "1.0.0". I figured it was better to have it fallback to the VersionPrefix in version.props than to produce "1.0.0"